sdk/storage/Azure.Storage.DataMovement/tests/JobPlanHeaderTests.cs

```diff
@@ -3,6 +3,7 @@
 
 using System.IO;
 using Azure.Storage.DataMovement.JobPlan;
+using FastSerialization;
```

Comment:
was this added by mistake?

Reply:
Yep. Stupid VS adding imports when you start typing stuff and it thinks it knows what you want.